packages/@ember/-internals/metal/lib/property_get.ts

```diff
@@ -90,6 +92,20 @@ export function get(obj: object, keyName: string): any {
     typeof keyName !== 'string' || keyName.lastIndexOf('this.', 0) !== 0
   );
 
+  if (DEBUG && !TOLERATE_DESTROYED) {
+    let objString: string;
+    TOLERATE_DESTROYED = true;
```

Comment:
added `TOLERATE_DESTROYED` flag, to avoid getting warning when `get` called within `toString`